<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yuv_player</class>
 <widget class="QMainWindow" name="yuv_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yuv player</string>
  </property>
  <widget class="Video_widget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="stereo_action"/>
    <addaction name="_exit_action"/>
   </widget>
   <widget class="QMenu" name="text_menu">
    <property name="title">
     <string>&amp;Text</string>
    </property>
    <addaction name="none_action"/>
    <addaction name="fc_action"/>
    <addaction name="tc_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="abt_action"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="mode_menu">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="y_action"/>
     <addaction name="u_action"/>
     <addaction name="v_action"/>
     <addaction name="separator"/>
     <addaction name="r_action"/>
     <addaction name="g_action"/>
     <addaction name="b_action"/>
     <addaction name="separator"/>
     <addaction name="rgb_action"/>
     <addaction name="grid_nyuv_action"/>
     <addaction name="grid_nrgb_action"/>
    </widget>
    <widget class="QMenu" name="stereo_menu">
     <property name="title">
      <string>Stereo</string>
     </property>
     <widget class="QMenu" name="menu_bfly">
      <property name="title">
       <string>Butterfly Split</string>
      </property>
      <addaction name="bhsleft_action"/>
      <addaction name="bhsright_action"/>
      <addaction name="bvsleft_action"/>
      <addaction name="bvsright_action"/>
     </widget>
     <widget class="QMenu" name="menu_split">
      <property name="title">
       <string>Normal Split</string>
      </property>
      <addaction name="nhsleft_action"/>
      <addaction name="nhsright_action"/>
      <addaction name="nvsleft_action"/>
      <addaction name="nvsright_action"/>
     </widget>
     <addaction name="left_action"/>
     <addaction name="right_action"/>
     <addaction name="add_action"/>
     <addaction name="sub_action"/>
     <addaction name="intleave_action"/>
     <addaction name="separator"/>
     <addaction name="menu_split"/>
     <addaction name="separator"/>
     <addaction name="menu_bfly"/>
    </widget>
    <addaction name="mode_menu"/>
    <addaction name="separator"/>
    <addaction name="stereo_menu"/>
    <addaction name="separator"/>
    <addaction name="screen_action"/>
    <addaction name="pbc_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="view_menu"/>
   <addaction name="text_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_action">
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/open/images/open_32.png</normaloff>:/open/images/open_32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="y_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/y/images/y.png</normaloff>:/y/images/y.png</iconset>
   </property>
   <property name="text">
    <string>Luma.Y</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="u_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/u/images/u.png</normaloff>:/u/images/u.png</iconset>
   </property>
   <property name="text">
    <string>Chroma.U</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="v_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/v/images/v.png</normaloff>:/v/images/v.png</iconset>
   </property>
   <property name="text">
    <string>Chroma.V</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="r_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/red/images/red.png</normaloff>:/red/images/red.png</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="g_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/green/images/green.png</normaloff>:/green/images/green.png</iconset>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="b_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/blue/images/blue.png</normaloff>:/blue/images/blue.png</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="rgb_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/rgb/images/rgb.png</normaloff>:/rgb/images/rgb.png</iconset>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="grid_nyuv_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nyuv/images/nyuv.png</normaloff>:/nyuv/images/nyuv.png</iconset>
   </property>
   <property name="text">
    <string>Normal.Y.U.V</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="abt_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="normal">
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="grid_nrgb_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nrgb/images/nrgb.png</normaloff>:/nrgb/images/nrgb.png</iconset>
   </property>
   <property name="text">
    <string>Normal.R.G.B</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="none_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fc_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/frames/images/calendar_32.png</normaloff>:/frames/images/calendar_32.png</iconset>
   </property>
   <property name="text">
    <string>Frame Count</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="tc_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/time/images/clock_32.png</normaloff>:/time/images/clock_32.png</iconset>
   </property>
   <property name="text">
    <string>Timecode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="screen_action">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="pbc_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Progress Bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="stereo_action">
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/open_stereo/images/open_stereo.png</normaloff>:/open_stereo/images/open_stereo.png</iconset>
   </property>
   <property name="text">
    <string>Open Stereo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="_exit_action">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionStereo">
   <property name="text">
    <string>Stereo</string>
   </property>
  </action>
  <action name="actionStereo_2">
   <property name="text">
    <string>Stereo</string>
   </property>
  </action>
  <action name="nhsleft_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nhsleft/images/split_32_hl.png</normaloff>:/nhsleft/images/split_32_hl.png</iconset>
   </property>
   <property name="text">
    <string>Horiz. Left</string>
   </property>
   <property name="iconText">
    <string>HL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="nvsleft_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nvsleft/images/split_32_vl.png</normaloff>:/nvsleft/images/split_32_vl.png</iconset>
   </property>
   <property name="text">
    <string>Vert. Left</string>
   </property>
   <property name="iconText">
    <string>VL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="add_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/add/images/plus_32.png</normaloff>:/add/images/plus_32.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="sub_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/minus/images/minus_32.png</normaloff>:/minus/images/minus_32.png</iconset>
   </property>
   <property name="text">
    <string>Sub</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="left_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/left/images/left_black_32.png</normaloff>:/left/images/left_black_32.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="right_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/right/images/right_green_32.png</normaloff>:/right/images/right_green_32.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="bhsleft_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/bhsleft/images/butterfly_32_hl.png</normaloff>:/bhsleft/images/butterfly_32_hl.png</iconset>
   </property>
   <property name="text">
    <string>Horiz. Left</string>
   </property>
   <property name="iconText">
    <string>HL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="bvsleft_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/bvsleft/images/butterfly_32_vl.png</normaloff>:/bvsleft/images/butterfly_32_vl.png</iconset>
   </property>
   <property name="text">
    <string>Vert. Left</string>
   </property>
   <property name="iconText">
    <string>VL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="intleave_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/interleave/images/mixture_32.png</normaloff>:/interleave/images/mixture_32.png</iconset>
   </property>
   <property name="text">
    <string>Interleave</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="nhsright_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nhsright/images/split_32_hr.png</normaloff>:/nhsright/images/split_32_hr.png</iconset>
   </property>
   <property name="text">
    <string>Horiz. Right</string>
   </property>
   <property name="iconText">
    <string>HR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="nvsright_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/nvsright/images/split_32_vr.png</normaloff>:/nvsright/images/split_32_vr.png</iconset>
   </property>
   <property name="text">
    <string>Vert. Right</string>
   </property>
   <property name="iconText">
    <string>VR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="bhsright_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/bhsright/images/butterfly_32_hr.png</normaloff>:/bhsright/images/butterfly_32_hr.png</iconset>
   </property>
   <property name="text">
    <string>Horiz. Right</string>
   </property>
   <property name="iconText">
    <string>HR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="bvsright_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="yuv_player.qrc">
     <normaloff>:/bvsright/images/butterfly_32_vr.png</normaloff>:/bvsright/images/butterfly_32_vr.png</iconset>
   </property>
   <property name="text">
    <string>Vert. Right</string>
   </property>
   <property name="iconText">
    <string>VR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Video_widget</class>
   <extends>QWidget</extends>
   <header location="global">video_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yuv_player.qrc"/>
 </resources>
 <connections/>
</ui>
